tools/harness-thci/ARM.py

```diff
@@ -2330,6 +2324,7 @@ def setCommisionerMode(self):
 
     def setPSKc(self, strPSKc):
         print '%s call setPSKc' % self.port
+        return
         try:
```

Comment:
In current implementation, we hope to input a key string as PSKc parameter, however, for harness, it will input a string(i.e. "threadecjpake"), this will cause fail to issue `dataset pskc <key string>`. Actually seems that we need to derive the PSKc from commissioning credential, so i keep this temporarily, will remove this later.